docs/gke/iap.md

```diff
@@ -57,46 +60,18 @@ The instructions below reference the following environment variables which you w
   * **IP_NAME** The name of the GCP static IP that you created above and will be associated with **DOMAIN**.
   * **NAMESPACE** The namespace where Kubeflow is deployed.
   * **ACCOUNT** The email address for your ACME account where certificate expiration notifications will be sent.
-
+  * **CLIENT_ID** The OAuth client ID obtained earlier.
+  * **CLIENT_SECRET** The OAuth client secret obtained earlier.
 
 ```
 ks generate cert-manager cert-manager --acmeEmail=${ACCOUNT}
 ks apply ${ENVIRONMENT} -c cert-manager
 
-ks generate iap-ingress iap-ingress --ipName=${IP_NAME} --hostname=${FQDN}
+ks generate iap-ingress iap-ingress --namespace=${NAMESPACE} --ipName=${IP_NAME} --hostname=${FQDN} --clientID=${CLIENT_ID} --clientSecret=${CLIENT_SECRET}
```

Comment:
nit: namespace is not needed as it is automatically inherited from ks env namespace.